yo1k/tic_tac_toe/core/game.py

```diff
@@ -45,53 +34,41 @@ class Cell:
     def __init__(self, x: int, y: int):
         assert 0 <= x < Board.size(), f"{x}"
         assert 0 <= y < Board.size(), f"{y}"
-        self.__x = x
-        self.__y = y
-
-    @property
-    def x(self) -> int:
-        return self.__x
-
-    @property
-    def y(self) -> int:
-        return self.__y
+        self.x = x
+        self.y = y
 
     def __str__(self):
         return (
             f"{type(self)}("
-            f"x={self.__x},"
-            f"y={self.__y})")
+            f"x={self.x},"
+            f"y={self.y})")
 
 
 class Board:
     def __init__(self, cells: Sequence[list[Optional[Mark]]] = None):
-        self.__cells = Board.__empty_cells() if cells is None else cells
-
-    def _set(self, cell: Cell, mark: Mark):
-        self.__cells[cell.x][cell.y] = mark
+        self.cells = Board.empty_cells() if cells is None else cells
 
-    def _get(self, cell: Cell) -> Optional[Mark]:
-        return self.__cells[cell.x][cell.y]
+    def set(self, cell: Cell, mark: Mark):
+        self.cells[cell.x][cell.y] = mark
 
-    def _clean(self):
-        self.__cells = Board.__empty_cells()
+    def get(self, cell: Cell) -> Optional[Mark]:
+        return self.cells[cell.x][cell.y]
 
-    @property
-    def cells(self):
-        return self.__cells
+    def clean(self):
+        self.cells = Board.empty_cells()
 
     @staticmethod
     def size() -> int:
         return 3
 
     @staticmethod
-    def __empty_cells() -> list[list[Optional[Mark]]]:
+    def empty_cells() -> list[list[Optional[Mark]]]:
```

Comment:
Well, there is no reason to make this static method public. This is unrelated to the idea of making all attributes public.

Reply:
Done.